cmd/jujud/agent/machine/stateworkers.go

```diff
@@ -41,12 +41,12 @@ func StateWorkersManifold(config StateWorkersConfig) dependency.Manifold {
 				return nil, err
 			}
 
-			st, err := stTracker.Use()
+			stPool, err := stTracker.Use()
```

Comment:
spTracker?
statepoolTracker?

Reply:
as discussed, no, StatePool is just an implementation detail. eventually this should be replaced with a different manager object which takes responsibility for state objects and their workers